Components/Alerts/Alerts.css

```diff
@@ -20,30 +19,9 @@
 
 .code-alert {
   height: 40vh;
-  margin: 2rem 5rem 2rem auto;
-  width: fit-content;
-}
-
-code {
-  font-family: inherit;
-}
-
-.code::-webkit-scrollbar {
-  width: 10px;
-}
-
-.code::-webkit-scrollbar-track {
-  background: hsl(var(--clr-white));
-}
-
-.code::-webkit-scrollbar-thumb {
-  background: hsl(var(--clr-dark));
+  width: 70%;
 }
 
 .alert-information span {
-  padding: 0.5rem;
-}
-
-.alert-information > * {
-  margin: 3rem;
+  padding: 0.125rem;
```

Comment:
Use px for this

Reply:
I guess it is fine, depends on every other person